mllib-dal/test.sh

```diff
@@ -53,7 +53,8 @@ suiteArray=(
   "classification.MLlibNaiveBayesSuite" \
   "regression.MLlibLinearRegressionSuite" \
   "stat.MLlibCorrelationSuite" \
-  "stat.MultivariateOnlineSummarizerSuite"
+  "stat.MultivariateOnlineSummarizerSuite" \
+  "oneDALSuite"
```

Comment:
if putting oneDALSuite in com.intel.oap.mllib,  we can't use Matrices.fromVectors function to convert Vector to Matrix, because fromVectors function was private[ml]. In my opinion, we can distinguish more fine-grained into ML and MLLIb likes following image.

![image](https://user-images.githubusercontent.com/45281494/154652897-fbaebf86-7254-4fc8-bc44-1d043bb2e8a9.png)

Reply:
I agree, thanks!